<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>498</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>40</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>휴먼엑스포</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>비트코인 정보 출력 프로그램</string>
    </property>
   </widget>
   <widget class="QLabel" name="trade_price">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>251</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>휴먼엑스포</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="high_price">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>150</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>150</y>
      <width>70</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>고가(당일):</string>
    </property>
   </widget>
   <widget class="QLabel" name="low_price">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>190</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>190</y>
      <width>70</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>저가:(당일):</string>
    </property>
   </widget>
   <widget class="QLabel" name="prev_closing_price">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>240</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>현재가격</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="closing_price">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>240</y>
      <width>70</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>전일종가:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>240</y>
      <width>70</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>최근거래량:</string>
    </property>
   </widget>
   <widget class="QLabel" name="trade_volume">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>240</y>
      <width>160</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>현재가격</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>70</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>24h거래량:</string>
    </property>
   </widget>
   <widget class="QLabel" name="acc_trade_volume_24h">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>150</y>
      <width>160</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>현재가격</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="acc_trade_price_24h">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>190</y>
      <width>160</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>현재가격</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_21">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>70</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>24h거래금액:</string>
    </property>
   </widget>
   <widget class="QLabel" name="signed_change_rate">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>90</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>변 화 율</string>
    </property>
   </widget>
   <widget class="QLabel" name="coin_ticker_label">
    <property name="geometry">
     <rect>
      <x>349</x>
      <y>80</y>
      <width>81</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>휴먼엑스포</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 127);</string>
    </property>
    <property name="text">
     <string>BTC</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>320</y>
      <width>70</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>* 알람가격</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>400</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>매도가격</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>400</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>매수가격</string>
    </property>
   </widget>
   <widget class="QPushButton" name="alarmButton">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>400</y>
      <width>75</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>알람시작</string>
    </property>
   </widget>
   <widget class="QComboBox" name="coin_comboBox">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>40</y>
      <width>130</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="coin_name">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>50</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>코인종류</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="alarm_price1">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>400</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="alarm_price2">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>400</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
